docs/development/room-dag-concepts.md

```diff
@@ -61,12 +65,11 @@ For example, when we fetch the event auth chain or state for a given event, we
 mark all of those claimed auth events as outliers because we haven't done the
 state calculation ourself.
 
+Outliers are sometimes referred to as floating outliers but there is no
```

Comment:
I've never heard them referred to as that...

Reply:
Maybe it was just a descriptor that came out of the intro discussions for discussing how MSC2716 could work. Can remove ⏩